example/service/s3/sync/sync.go

```diff
@@ -69,10 +70,19 @@ func (iter *SyncFolderIterator) UploadObject() s3manager.BatchUploadObject {
 		iter.err = err
 	}
 
+	fileSplit := strings.Split(fi.key, ".")
+	extension := "." + fileSplit[len(fileSplit)-1]
```

Comment:
It would be better to use [filepath.Ext](https://godoc.org/path/filepath#Ext) here instead of doing the split manually.